configs/siklu_80X0_defconfig

```diff
@@ -71,7 +71,7 @@ BR2_PACKAGE_NTP_NTPDC=y
 BR2_PACKAGE_NTP_NTPTIME=y
 BR2_PACKAGE_SOCAT=y
 BR2_PACKAGE_TCPDUMP=y
-BR2_PACKAGE_WIRELESS_TOOLS=y
+#BR2_PACKAGE_WIRELESS_TOOLS=y
```

Comment:
Need to change configuration using buikdroot interactive menu and let him to modify the config (instead of modifying the config file by yourself)

Reply:
we don't use .config -- it is generated file, it based on top of this defconfig -- so this one is template for .config

Reply:
Exactly, you don't need to modify generated file by hands